tests/test_claim_material_law_rescue.py

```
@@ -0,0 +1,231 @@
+        "Подготовить ответ на обращение и проверить реквизиты документа без денежного требования.",
+        research,
+    )
+
+    assert result.verified_claims == before
+    assert opened is False
+
+
+def test_unverified_claims_keep_needs_verification_after_successful_rescue(monkeypatch) -> None:
+    corpus = _fresh_corpus()
+    _install_corpus(monkeypatch, corpus)
+    research = _research(unverified=["Не подтверждено дополнительное требование."])
```

Ruff (0.16.2)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)